force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +350,17 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
-
+        Integer wordIndex = -1;
         // Loop through the list of words
             // If the current word is the word to find, exit the loop
+        for (Integer i = 0; i < words.size(); i++){
+            if (words[i] == wordToFind) {
+                wordIndex = i;
+                break;
```

Comment:
I would say you don't even need the break here. If word[i] == wordToFind then just return i;
 public static Integer findWordInList(String wordToFind, List<String> words) {
     for(Integer i = 0; i < words.size(); i++){
         if(words[i] == wordToFind){
             return i;
         }
     }
        return -1; // Replace null with the variable you used to store the resulto
    }